DigitalPassaport/DigitalPassaport/Epics/CreateAccount/Repository/CreateAccountRepository.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  CreateAccountRepository.swift
+//  DigitalPassaport
+//
+//  Created by Milena Predic on 24.8.23..
+//
+
+import Foundation
+import Combine
+import Alamofire
+
+protocol CreateAccountRepositoryType {
+    func triggerApi()
+}
+
+class CreateAccountRepository: CreateAccountRepositoryType {
+    
+    private let networkingService: NetworkingService
+    private var passes: [Pass] = []
+    private var user: User?
+    private var cancellables: Set<AnyCancellable> = []
+    
+    init(networkingService: NetworkingService) {
+        self.networkingService = networkingService
+    }
+
+    func triggerApi() {
+        let url = URL(string: APIEndpoints.account)!
+        networkingService.post(url, parameters: [:])
+            .sink(receiveCompletion: { completion in
+                // Handle completion if needed
+            }, receiveValue: { [weak self] data in
+                do {
+                    let jsonObject = try JSONSerialization.jsonObject(with: data, options: [])
```

Comment:
It shouldn't do here, you have to parse it inside a struct by overriding decoder, something like you did for interview task